simplified-profiles-api/src/main/java/org/nypl/simplified/profiles/api/ProfilePreferences.kt

```diff
@@ -43,5 +43,8 @@ data class ProfilePreferences(
 
   /** @return `true` if the debug settings should be visible. */
 
-  val showDebugSettings: Boolean = false
+  val showDebugSettings: Boolean = false,
+
+  /** @return `true` if user selects SimplyE collection. */
+  val requiresAge: Boolean = false,
```

Comment:
I'm not sure this flag is useful.

`(authentication is AccountProviderAuthenticationDescription.COPPAAgeGate ||
        buildConfig.showAgeGateUi) && preferences.dateOfBirth == null` is a good criterion, isn't it?

Reply:
Sorry, I made a stupid copy/paste from your code. The right criterion is likely to be `authentication is AccountProviderAuthenticationDescription.COPPAAgeGate` **&&** `buildConfig.showAgeGateUi && preferences.dateOfBirth == null`. That should fix your issue.